packages/riipen-ui/src/components/Input.jsx

```diff
@@ -122,7 +122,7 @@ class Input extends React.Component {
             font-family: ${theme.typography.fontFamily};
             font-size: 14px;
             outline: none;
-            padding: ${theme.spacing(2)}px;
+            padding: ${theme.spacing(1.5)}px;
```

Comment:
the original switch to the spacing stuff was because maribel would give us things like spacing s1 s2 s3 which were 5 10 15 etc. Dont know if we want to be doing fractions here? (unless maribel commented on the spacing)